### DISCUSSION TITLE
Dev yearly wrapped

### DISCUSSION THREAD

**Discussion by pdicerbo** [dailydotdev/daily#1637] [Feature Requests]
I just completed the survey and I had an idea of a possible feature.
It would be a kind of cool to have a yearly report about what we read, similarly to what Spotify does in this period of the year :)

What do you think about this?

**Comment by idoshamun** [comment]
Hi @pdicerbo,

Every year, we consider if we should pursue this idea, but unfortunately, we never get to prioritize it.
I hope next year will be different 😬

Thanks a lot for the idea
